client/src/tutorials/Lumberjack/pages/3.md

```diff
@@ -0,0 +1,110 @@
+## Js client
+
+(client.ts file)
+The client you can run by typing "run" in the terminal.
+This will create a new account if it does not yet exist and then chop a tree and print out the wood and energy value of the player.
+
+```js
+let gameDataAccount = await pg.program.account.playerData
+  .fetch(playerDataPda)
+  .catch(async (error) => {
```

Comment:
This is an anti-pattern, please use `try { ... } catch { ... }` sytnax instead of awaiting a `Promise.catch`